cmd/hatchet-api/api/run.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func Start(cf *loader.ConfigLoader, interruptCh <-chan interface{}, version string) error {
 	// init the repository
-	configCleanup, sc, err := cf.LoadServerConfig(version)
+	configCleanup, sc, err := cf.CreateServerFromConfig(version)
```

Comment:
```suggestion
	configCleanup, server, err := cf.CreateServerFromConfig(version)
```

nit + not important, but if we are doing a quick pass for consistency this would match the other files

Reply:
agreed

Reply:
@reillyse sorry for the confusion, this does lead to a compile error with just this change